Header header
float32 cell_width
float32 cell_height
int32 referance               #position of referance cell
geometry_msgs/Point[] cells
uint16[] ids
